### PR TITLE
Added docs on how to integrate JobSet

### LINKED ISSUE #738: Support JobSet in Kueue
**What would you like to be added**:

The ability to control JobSet startup through Kueue.

**Why is this needed**:

To support a wide variety of workloads

**Completion requirements**:

This enhancement requires the following artifacts:

- [ ] Design doc
- [ ] API change
- [ ] Docs update

The artifacts should be linked in subsequent comments.

### PR DESCRIPTION
**Pull request by mcariatm**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
Documenting JobSet support

#### Which issue(s) this PR fixes:

Fixes #738 

#### Special notes for your reviewer:

```release-note
NONE
```

### CODE REVIEW

**Comment by stuton** [comment]
/retitle Added docs on how to integrate JobSet

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve
/hold
for #738

**Comment by trasc** [comment]
Add a section describing https://github.com/kubernetes-sigs/kueue/pull/762/files#r1237461450

File: site/content/en/docs/tasks/run_jobsets.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+```yaml
+metadata:
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.replicatedJobs`.
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/run_jobsets.md:45]: Also mention the replicas

File: site/content/en/docs/tasks/run_jobsets.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+```yaml
+metadata:
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.replicatedJobs`. The first [PriorityClassName](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass) of `spec.replicatedJobs` that is not empty will be used as the priority.
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/run_jobsets.md:45]: Priority is not directly related to resources.

Please do it in a separate heading.

Also don't forget to mention the number of replicas with regards to total resource calculations.

File: site/content/en/docs/tasks/run_jobsets.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.replicatedJobs`. Should also be taken into account that number of replicas, parallelism and completions affect the resource calculations. 
+
+```yaml
+    - template:
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/run_jobsets.md:48]: add replicas here

File: site/content/en/docs/tasks/run_jobsets.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+```yaml
+metadata:
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.replicatedJobs`. Should also be taken into account that number of replicas, parallelism and completions affect the resource calculations.
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/run_jobsets.md:45]: oh, I forgot to ask to mention parallelism :(

**Comment by alculquicondor** [comment]
/release-note-edit
```release-note
NONE
```

**Pull request merged**